firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -229,6 +229,7 @@ String getName() {
   @NonNull
   @Override
   public Task<InstallationTokenResult> getToken(boolean forceRefresh) {
+    preConditionChecks();
```

Comment:
what about deleteToken?

Reply:
We never had it on deleteToken. Shall we add it?